src/test/java/net/openhft/chronicle/queue/MappedFileUtil.java

```diff
@@ -0,0 +1,64 @@
+package net.openhft.chronicle.queue;
+
+import net.openhft.chronicle.core.Jvm;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public enum MappedFileUtil {
```

Comment:
We could promote this to Chronicle-Test-Framework if it proves reliable. It seems like it might be useful in other contexts.